.github/actions/create-featurenet/action.yml

```diff
@@ -148,21 +148,24 @@ runs:
           pr_image_tag="fe-${{ steps.get-ref-properties.outputs.branch-name-for-argo-with-sha }}"
           fnet_aleph_node_image="${{ inputs.ecr-public-registry }}feature-env-aleph-node:${pr_image_tag}"
           fnet_bootstrap_chain_node_image="${{ steps.get-ref-properties.outputs.sha }}"
+          fnet_create_hook="false"
         elif [[ "${{ inputs.featurenet-aleph-node-image }}" == "testnet" || \
             "${{ inputs.featurenet-aleph-node-image }}" == "mainnet" ]]; then
           ecr_image_tag=$(echo "${{ steps.get-node-commit-sha.outputs.sha }}")
```

Comment:
Why do we have that line?  Can't we just pass `${{ steps.get-node-commit-sha.outputs.sha }}` without `echo`-ing it into `ecr_image_tag`?  I mean line below.